Events/src/main/java/com/github/swcraftlyon/meetup/infra/ApplicationConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.github.swcraftlyon.meetup.infra;
+
+
+public class ApplicationConfig {
+
+    public static void getStartupMessage(){
+        System.out.println("MMMMMMMM               MMMMMMMM                                                 tttt                                                     !!! ");
```

Comment:
No opinion about keep or throw away but if we keep it : if it's a get, than it guess it should return the banner ? What about spliting the banner definition and it's system output ?